client/packages/requisitions/src/RnRForms/DetailView/RnRFormLine.tsx

```diff
@@ -114,23 +114,30 @@ export const RnRFormLine = ({
       ? theme.palette.text.disabled
       : theme.palette.text.primary;
 
+  const readOnlyBackgroundColor = theme.palette.background.drawer;
```

Comment:
I had an issue with some of my code, making recursive renders, i notice that styles were being constantly regenerated an inserted, now I am looking sideways at inline styles in MUI haha.

https://drive.google.com/drive/u/1/folders/1_JLn9oSkjvdnLuecANK1E3t-l8eQBBGy
And issue: https://github.com/msupply-foundation/open-msupply/issues/7643